# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
#goal definition
bool goal_mode
al_msgs/JointCmd[] goal_joint_pos
geometry_msgs/Pose goal_cartesian_pos
